doc/SEMANTICS.md

```diff
@@ -270,4 +275,5 @@ Mountpoint provides strong read-after-write consistency for new object creation
 * A process deletes an existing object from your S3 bucket using another client, and then tries to open the object with Mountpoint and read from it. The open operation will fail.
 * A process deletes an existing object from your S3 bucket, using either Mountpoint or another client, and then lists the directory the object was previously in with Mountpoint. The object will not appear in the list.
 * A process deletes an existing object from your S3 bucket using another client, and then queries the object’s metadata with Mountpoint using the `stat`` system call. The returned metadata could reflect the old object for up to 1 second after the DeleteObject request.
+* A process opens a file with Mountpoint, and another Mountpoint client on another instance opens the same file. In this case, the behaviour is determined by the [S3 semantics for concurrent multipart uploads](https://docs.aws.amazon.com/AmazonS3/latest/userguide/mpuoverview.html#distributedmpupload).
```

Comment:
I'd not include this, or we need to get into a lot of detail and potentially consider single PutObject as well.

Ideally we'd have an S3 User Guide page covering the different upload methods and sequencing, but I'm not aware of one right now.

Reply:
Dropped it.